File: src/ZKWeb.MVVMPlugins/MVVM.Common.Organization/src/Application/Dtos/TenantOutputDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Text;
using ZKWeb.MVVMPlugins.MVVM.Common.Base.src.Application.Dtos;

namespace ZKWeb.MVVMPlugins.MVVM.Common.Organization.src.Application.Dtos {
	[Description("租户传出信息")]
	public class TenantOutputDto : IOutputDto {
		public Guid Id { get; set; }
		public string Name { get; set; }
		public bool IsMaster { get; set; }
		public string CreateTime { get; set; }
		public string UpdateTime { get; set; }
		public string Remark { get; set; }
	}
}

File: src/ZKWeb.MVVMPlugins/MVVM.Angular.Website/static/src/modules/generated_module/dtos/tenant-output-dto.ts
/** 租户传出信息 */
export class TenantOutputDto {
	/** 租户Id */
	public Id: string;
	/** 租户名称 */
	public Name: string;
	/** 是否主租户 */
	public IsMaster: boolean;
	/** 创建时间 */
	public CreateTime: string;
	/** 更新时间 */
	public UpdateTime: string;
	/** 备注 */
	public Remark: string;
}
